### PR TITLE
Ray.intersectSegment method added

### PR DESCRIPTION
**Pull request by nockawa**

### CODE REVIEW

File: src/Culling/babylon.ray.ts [hunk 1]
```diff
@@ -192,6 +192,91 @@
...
+
+            // get the difference of the two closest points
+            var dP = w.add(u.multiplyByFloats(sc, sc, sc)).subtract(v.multiplyByFloats(tc, tc, tc));  // = S1(sc) - S2(tc)
+
+            var isIntersected = dP.lengthSquared() < (threshold * threshold);   // return intersection result
+
+            if (isIntersected) {
+                return tc;
+            }
+            return undefined;
```
Comment (RaananW) [inline src/Culling/babylon.ray.ts]: maybe returning -1 will be better?

**Comment by nockawa** [comment]
The current commit has an incomplete version of the method, I don't check if the intersection is behind or further the ray.
For the return data, I think returning null in case of no intersection is good. I don't think we should use intersectionInfo considering the current state of the class.

**Comment by nockawa** [comment]
Am i supposed to update the what's new for such small changes?

**Comment by deltakosh** [comment]
yes please..all changes should be documented

**Comment by nockawa** [comment]
I'll do it altogether the next pr then

**Pull request merged**